pandas/plotting/_matplotlib/core.py

```diff
@@ -106,6 +106,7 @@ def __init__(
         colormap=None,
         table=False,
         layout=None,
+        include_bool=False,
```

Comment:
This feels better, thanks. Will need to be documented in PlotAccessor.

Reply:
thanks and added doc! @TomAugspurger